db/migrate/20230109080950_rename_solidus_paypal_braintree_source_type.rb

```diff
@@ -0,0 +1,16 @@
+class RenameSolidusPaypalBraintreeSourceType < ActiveRecord::Migration[6.1]
+  def up
+    Spree::Payment.where(source_type: 'SolidusPaypalBraintree::Source')
+      .update_all("source_type = 'SolidusBraintree::Source'")
+
+    Spree::WalletPaymentSource.where(payment_source_type: 'SolidusPaypalBraintree::Source')
+      .update_all("payment_source_type = 'SolidusBraintree::Source'")
+
+    Spree::PaymentMethod.where('type = ?', 'SolidusPaypalBraintree::Gateway')
+      .update_all("type = 'SolidusBraintree::Gateway'")
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
```

Comment:
Not allowing the data migration to be reverted to be safe: it's best not to assume what data previously existed in the user's database before the migration was ran.